### ISSUE TITLE
evaluate_generator broken in 2.3

### ISSUE THREAD

**Issue by eyaler** [keras-team/keras#13469]
evaluate_generator will silently return the loss computed only on the first batch and not aggregate over batches. other metrics are ok, the problem is only with the loss.
it was ok in 2.2.5. and broken for 2.3.0, 2.3.1 due to this commit: https://github.com/keras-team/keras/commit/b938b91b527760cef3af29031e74ab181f2fc2b9
@pavithrasv

**Comment by eyaler** [comment]
@gowthamkpr thanks but this is not a performance issue - this is a serious regression bug in evaluating the model (and in training if you use early stopping)

**Comment by pavithrasv** [comment]
Thank you for reporting this issue @eyaler. If you know the fix for the issue, please feel free to send us a PR. Otherwise, I will look into the fix as soon as possible.

**Comment by eyaler** [comment]
colab has not update from 2.2.5 to 2.3.1 and this will cause trouble

**Comment by eyaler** [comment]
can anybody confirm if this has been fixed?

**Comment by eyaler** [comment]
this is still a major issue. why is this closed?
